shell/platform/common/client_wrapper/include/flutter/encodable_value.h

```diff
@@ -110,6 +110,7 @@ using EncodableValueVariant = std::variant<std::monostate,
                                            std::vector<double>,
                                            EncodableList,
                                            EncodableMap,
+                                           std::vector<float>,
```

Comment:
This violates the guarantees made by this API; see lines 99-100 and 146-147.

Please fix or revert this ASAP.